code/game/objects/items/holy_weapons.dm

```diff
@@ -452,7 +452,6 @@
 		S.ckey = C.ckey
 		S.fully_replace_character_name(null, "The spirit of [name]")
 		S.status_flags |= GODMODE
-		S.language_holder = user.language_holder.copy(S)
```

Comment:
No replacement here ?

Reply:
This was in the known issues section, back in now. :)